dev-python/sentry-sdk/files/sentry-sdk-0.14.4-disable-auto-enabling-intregrations-tests.patch

```diff
@@ -0,0 +1,24 @@
+--- a/tests/test_basics.py
++++ b/tests/test_basics.py
+@@ -39,21 +39,6 @@
+
+     assert event["exception"]["values"][0]["value"] == "aha! whatever"
+
+-
+-def test_auto_enabling_integrations_catches_import_error(sentry_init, caplog):
```

Comment:
Sed it out via `sed -e 's:test_auto_blahblah:_&:' -i ...`

Reply:
Fixed